timelock-impl/src/main/java/com/palantir/atlasdb/timelock/auth/config/AdminPrivilegesConfiguration.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.auth.config;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.palantir.atlasdb.timelock.auth.api.ClientId;
+import com.palantir.atlasdb.timelock.auth.api.Privileges;
+
+@JsonSerialize(as = ImmutableAdminPrivilegesConfiguration.class)
+@JsonDeserialize(as = ImmutableAdminPrivilegesConfiguration.class)
+@JsonTypeName(AdminPrivilegesConfiguration.TYPE)
+@Value.Immutable
+public abstract class AdminPrivilegesConfiguration implements PrivilegesConfiguration {
+    static final String TYPE = "admin";
+
+    @JsonProperty("client-id")
+    @Override
+    public abstract ClientId clientId();
+
+    @Value.Derived
+    @Override
+    public Privileges privileges() {
+        return Privileges.ADMIN;
+    }
+}
```

Comment:
I guess we don't have a good way to protect against rogue clients making themselves admin, but in any case lasting changes to this can only be made through deployment infra tools, and the working assumption is that if those are broken, so is the system